src/BuildASub.java

```diff
@@ -60,4 +69,60 @@ public static String chooseMeat() throws Exception {
             throw new Exception("Meat not set!");
         }
     }
+
+    public static Boolean addSalad (){
```

Comment:
spaces should be consistent across method names, again this is down to Java style. `addSalad (){` should be `addSalad() {`